src/Format/ItemList.php

```diff
@@ -41,6 +41,7 @@ public function links()
 
         $append_plain  = $this->getPlainTextExtras($extras, 'append_plain', $this->items);
         $prepend_plain = $this->getPlainTextExtras($extras, 'prepend_plain', $this->items);
+        $attr          = array_key_exists('attr', $extras) ? $extras['attr'] : '';
```

Comment:
Using array_get seems a little cleaner?
```
$attr = array_get($extras, 'attr', '');
```

Reply:
@jterrono Yeah, I was a little concern to use that as it's too tie to Laravel when considering templater isn't fully Laravel. But realized we do pull the `illuminate/support`, so I assume we free to use all those helpers. Will update.